.vscode/launch.json

```diff
@@ -8,6 +8,7 @@
             "module": "kodi_addon_checker.__main__",
             "args": [
                 "--branch=BRANCH_NAME",
+                "--PR"
```

Comment:
Remove this line, it's a repo run here.

Reply:
you commented on the wrong line
the one you meant was removed

Reply:
everything good, just add the comma after this line and we can merge

Reply:
Done